pkg/apiserver/openapi/zz_generated.openapi.go

```diff
@@ -703,31 +727,6 @@ func schema_pkg_apis_kappctrl_v1alpha1_AppFetchHelmChart(ref common.ReferenceCal
 	}
 }
 
-func schema_pkg_apis_kappctrl_v1alpha1_AppFetchHelmChartRepo(ref common.ReferenceCallback) common.OpenAPIDefinition {
```

Comment:
Related to comment [here](https://github.com/vmware-tanzu/carvel-kapp-controller/pull/506/files#r798099253).